crates/bevy_render/src/entity.rs

```diff
@@ -77,3 +86,40 @@ impl Default for Camera2dBundle {
         }
     }
 }
+
+/// A component bundle for "orthographic camera" entities
+///
+/// Uses a normalized orthographic projection.
+///
+/// Use this for CAD-like 3D views or non-pixel-oriented 2D games.
+#[derive(Bundle)]
+pub struct CameraOrthoBundle {
```

Comment:
That is a very long name, and the camera is useful for 2D games, too. Actually, I'd advise anyone who is making a 2D game, but not going for a pixel-perfect artstyle, to consider if they would be better served by this new projection. It would automatically take care of scaling and window resizing / different resolutions, without having to mess with the camera.

I'd prefer if the name doesn't include either "2D" or "3D".